rskj-core/src/main/java/co/rsk/rpc/modules/txpool/TxPoolModuleImpl.java

```diff
@@ -0,0 +1,54 @@
+package co.rsk.rpc.modules.txpool;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import org.ethereum.core.PendingState;
+import org.ethereum.facade.Ethereum;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TxPoolModuleImpl implements TxPoolModule{
+
+    private Ethereum eth;
```

Comment:
`Prefer private final fields for dependencies`: https://github.com/rsksmart/rskj/blob/contributing/CONTRIBUTING.md#java-styleguide

Reply:
Maybe you can also be the first one to apply the `Check not null preconditions` styleguide!